notebooks/5-stellar-flares/interm_notebook_5_classify.ipynb

```diff
@@ -0,0 +1,1428 @@
+{
```

Comment:
<p>We should talk a little bit about loss, accuracy, and what they mean in the context of ML. If something has gone wrong in our model fitting, how would we expect the plots below to behave?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>